lib/internal/fs/promises.js

```diff
@@ -276,12 +276,9 @@ class FileHandle extends EventEmitter {
   /**
    * @typedef {import('../webstreams/readablestream').ReadableStream
    * } ReadableStream
-   * @param {{
-   *   type?: string;
-   *   }} [options]
    * @returns {ReadableStream}
    */
-  readableWebStream(options = kEmptyObject) {
+  readableWebStream(options = { type: 'bytes' }) {
```

Comment:
I think this needs to be null prototyped

Reply:
I'm grepping for usages of `ObjectCreate` and don't really see anyone else doing that. So, are you sure, and if so, is there some other way than ObjectCreate?  Can you be specific with what you want me to do?

Reply:
Add a __proto__:null to the object